equinix/resource_network_device_link.go

```diff
@@ -20,23 +20,27 @@ import (
 )
 
 var networkDeviceLinkSchemaNames = map[string]string{
-	"UUID":      "uuid",
-	"Name":      "name",
-	"Subnet":    "subnet",
-	"Devices":   "device",
-	"Links":     "link",
-	"Status":    "status",
-	"ProjectID": "project_id",
+	"UUID":           "uuid",
+	"Name":           "name",
+	"Subnet":         "subnet",
+	"Devices":        "device",
+	"Links":          "link",
+	"MetroLinks":     "metro_link",
+	"RedundancyType": "redundancy_type",
+	"Status":         "status",
+	"ProjectID":      "project_id",
 }
 
 var networkDeviceLinkDescriptions = map[string]string{
-	"UUID":      "Device link unique identifier",
-	"Name":      "Device link name",
-	"Subnet":    "Device link subnet CIDR.",
-	"Devices":   "Definition of one or more devices belonging to the device link",
-	"Links":     "Definition of one or more, inter metro connections belonging to the device link",
-	"Status":    "Device link provisioning status",
-	"ProjectID": "The unique identifier of Project Resource to which device link is scoped to",
+	"UUID":           "Device link unique identifier",
+	"Name":           "Device link name",
+	"Subnet":         "Device link subnet CIDR.",
+	"Devices":        "Definition of one or more devices belonging to the device link",
+	"Links":          "Definition of one or more, inter metro connections belonging to the device link",
+	"MetroLinks":     "Definition of one or more, inter or intra metro connections belonging to the device link",
+	"RedundancyType": "Device link redundancy type",
```

Comment:
nit: the descriptions should be consistent with the markdown (https://github.com/equinix/terraform-provider-equinix/pull/621/files?diff=unified&w=1#diff-4c818a92bfcd55907ea7e51255ff4c04d63a494a1f0dd6e3b689388b11f1a350R53) to ease the path to #201